### PR TITLE
Add details popup window to the project card button

### PR DESCRIPTION
**Pull request by tasnimchy96**
In this branch we:

- Added details popup window to the project card button and made it responsive for mobile and desktop

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -49,6 +49,207 @@ header {
...
+  margin: 1rem 0;
+  border: 0.063rem solid #ebecf0;
+  width: 100%;
+}
+
+@media only screen and (min-device-width: 768px) {
+  /* .my-modal > img {
+    width: auto;
+    height: 40rem;
+  } */
```
Comment (Ghiftee) [inline style.css]: - [ ] Kindly remove commented codes that are not descriptive in nature as they make your codebase messy

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @tasnimchy96 and @melybee-07  👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ App runs as expected
✔️ Linters are set up correctly
✔️ Design matches given template

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
- [x] Your license in your README is currently a template, kindly customize it. This would make your documentation more professional.
- [x] Good job following the specifics of the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-418&t=xZbg2RGg4YJzwpMl-0), however, your project should look exactly like the given design so kindly fix the differences highlighted below:
### Your design
- The spacing at the end of your design does not match the one at the end of the figma design. 
Screenshot 2023-03-18 at 08 02 10 (2)

### Figma design
Screenshot 2023-03-18 at 08 12 06 (2)

### Your design
- You are currently using the wrong colors for these elements
Screenshot 2023-03-18 at 08 03 19 (2)

### Figma design
Screenshot 2023-03-18 at 08 03 36 (2)

### Your design
- You are also using the wrong color and background color for this element
Screenshot 2023-03-18 at 08 04 06 (2)




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css (hunk 1 above)
Comment (tasnimchy96) [reply style.css]: Done

**Review by joel-kalema — APPROVED** [review]
Hi @tasnimchy96 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**